sources/packages/web/src/types/contracts/student/ApplicationContract.ts

```diff
@@ -138,11 +144,9 @@ export interface StudentApplicationDetailsForTracking {
 /**
  * Interface for application summary
  */
-export interface ApplicationSummaryDTO {
-  applicationNumber: string;
+export interface ApplicationSummaryDTO extends ApplicationPrimaryDTO {
```

Comment:
I am missing why GetApplicationBaseDTO is now extending the ApplicationPrimaryDTO. Can you please clarify it?